src/rlx_assemble.erl

```diff
@@ -26,11 +27,28 @@ do(Release, State) ->
         andalso rlx_state:mode(State1) =/= dev of
         true ->
             ?log_debug("Stripping release beam files", []),
-            case beam_lib:strip_release(OutputDir) of
+            %% make *.beam to be writeable for strip.
+            ModeChangedFiles = [ 
+                OrigFileMode
+            || File <- rlx_file_utils:wildcard_paths([OutputDir ++ "/**/*.beam"]), 
+                OrigFileMode <- begin
+                    {ok, #file_info{mode = OrigMode}} = file:read_file_info(File), 
+                    case OrigMode band 8#0200 =/= 8#0200 of
+                        true ->
+                            file:change_mode(File, OrigMode bor 8#0200),
+                            [{File, OrigMode}];
+                        false -> []
+                end
```

Comment:
nitpick, looks like this indentation is off a few spaces, should line up under the `case`.

Reply:
Oops. Fixed now.